File: C/13.c
#include <stdio.h>

int main()
{
    int arr[7][5] = {
        {10, 20, 30, 40},
        {100, 200, 300}
    };

        arr[1][3] = 400;
        arr[2][2] = 2000;
    
    int row = sizeof(arr) / sizeof(arr[0]);
    int col = sizeof(arr[0]) / sizeof(arr[0][0]);
    printf("row = %d",row);
    printf("\ncol = %d",col);

    printf("\n-----------\n");

    for (int i = 0; i < row; i++){
        for (int j = 0; j < col; j++)
        {
            printf("%d ", arr[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: python/13.py
data = [10,20,30,40,41,83,80,5,3]

# print(data)

# for i in data:
#     print(i, end=" ") 

# even = 0
# odd = 0

# for i in data:
#     if i % 2 == 0:
#         even = even + 1
#     else:
#         odd = odd + 1

# print("Even: ", even)
# print("Odd: ", odd)


data = [
    [1,2,3],
    [10,20,30],
    [111,222,333]
]

print(data[1][1])

for row in data:
    for col in row:
        print(col, end=" ")
    print()